app/controllers/finance/api/invoices_controller.rb

```diff
@@ -21,7 +21,7 @@ def index
   # GET /api/invoices/{id}.xml
   def show
     respond_with(invoice) do |format|
-      format.pdf { redirect_to invoice.pdf.url }
+      format.pdf { redirect_to invoice.pdf.expiring_url }
```

Comment:
One small issue here is the link in the UI will expire after an hour unless the page is refreshed.

![image](https://github.com/user-attachments/assets/42e9b218-07f8-4632-9bff-fc2bb8c0f541)

Reply:
This is how I think it works with S3 storage so I guess that is an acceptable consequence.